features/steps/invitation_steps.js

```diff
@@ -36,3 +59,44 @@ Then(
     assert(await page.hasInvitation({ invitation, status }));
   }
 );
+
+Then(
+  "{a} {actor} becomes {a} {actor} of {a} {space}",
+  /**
+   * @param {Actor} actor
+   * @param {Actor} role
+   * @param {Space} space
+   */
+  function (_, actor, _2, role, _3, space) {
+    return (
+      actor
+        .signIn(this.driver, space)
+        // @todo Refactor to live in the harness
+        .then(() =>
+          new Component(
+            this.driver,
+            'header a.--configure[aria_label="Configure Space"]'
+          ).isDisplayed()
+        )
+        .then((displayed) => assert(displayed))
+    );
+  }
+);
+
+Then(
+  "all other Invitations to {actor} for {a} {space} no longer have {a} status of {string}",
+  /**
+   * @param {Actor} actor
+   * @param {Space} space
+   * @param {string} status
+   */
+  function (actor, a, space, a2, status) {
```

Comment:
Seems like `a` and `a2` params here be underscores instead of `a`s, to be consistent with what we have on line 70, and to make it clearer that those are unused.

Reply:
Good catch! My robot is supposed to get those but I did not.